packages/server/config/clientIndexPath.ts

```diff
@@ -0,0 +1,11 @@
+import path from "path";
+import { fileURLToPath } from "url";
+
+const clientIndexPath = () => {
+  const currentPath = path.dirname(fileURLToPath(import.meta.url));
+  return process.env.NODE_ENV !== "development"
+    ? path.join(currentPath, "../public/dist/index.html")
+    : path.join(currentPath, "../../client/public/index.html");
+};
```

Comment:
We're telling the server where to find the files for the client, depending on whether we want the hot(slower than the real client devServer)-reloaded `development` version or the bundled `production` one.